content/cloud/index.md

```diff
@@ -29,5 +29,5 @@ If you have any questions (and cannot find answers) on this site, please go to o
 
 The Fermyon Cloud is currently an open beta service. This means that Fermyon Technologies does not provide any service-level agreements on the service, including the workloads you choose to deploy. We also reserve the right to break compatibility with:
 
-- the Fermyon Cloud [API](rest-api), and
-- support for previous and current versions of the [Spin SDK](/spin)
+- The Fermyon Cloud [API](rest-api), and
```

Comment:
I realise you are only changing the capitalisation here but the phrasing here puzzles me.  The Fermyon Cloud might break compatibility with the Fermyon Cloud?

Reply:
Thanks, I have updated this in the new push.